index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>Jose Delphin Portfolio</title>
+  <link href="https://fonts.googleapis.com/css2?family=Inter&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="./styles.css">
+</head>
+
+<body>
+  <div class="background_desktop">
+    <header class="navigation">
+      <a class="logo" href="#">Welcome</a>
```

Comment:
- [ Required ] Good job adding the logo as a link :clap: :clap: , however you forgot to put a link in the `href` attribute of the `a` tag. Please consider adding a link to the `index.html` file, so that users of your webpage can use it to navigate to the home page.
### Example code
```HTML
<a href="index.html" class="logo"> Welcome </a>
```

Reply:
Understood. 
Thank you.